pkg/plugins/mellanox/mellanox_plugin.go

```diff
@@ -46,8 +58,11 @@ func init() {
 	Plugin = MellanoxPlugin{
 		PluginName:  "mellanox_plugin",
 		SpecVersion: "1.0",
+		nicSyncList: &nicSpecMap{},
 	}
 	mellanoxNicsStatus = map[string]map[string]sriovnetworkv1.InterfaceExt{}
+	// Launch periodic check for VF admin mac and node guid workers
+	go wait.Forever(Plugin.nicSpecPeriodicCheck, 30*time.Second)
```

Comment:
why not in the OnNodeStateChange?

Reply:
`OnNodeStateChange` is called every time a policy is changed which needs to maintain that no other goroutines are running but init is called once in all process runtime